nrich-excel/src/test/java/net/croz/nrich/excel/util/TypeDataFormatUtilTest.java

```diff
@@ -0,0 +1,70 @@
+package net.croz.nrich.excel.util;
+
+import net.croz.nrich.excel.api.model.TypeDataFormat;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZonedDateTime;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class TypeDataFormatUtilTest {
+
+    @Test
+    void shouldReturnTypeDataFormatList() {
+        // given
+        String dateFormat = "dd.MM.yyyy.";
+        String dateTimeFormat = "dd.MM.yyyy. HH:mm";
+        String integerFormat = "#,##0";
+        String decimalFormat = "#,##0.00";
+
+        // when
+        List<TypeDataFormat> formatList = TypeDataFormatUtil.resolveTypeDataFormatList(dateFormat, dateTimeFormat, integerFormat, decimalFormat, true, null);
+
+        // then
+        assertThat(formatList).extracting("type").containsExactly(Date.class, Instant.class, LocalDate.class, LocalDateTime.class, ZonedDateTime.class, OffsetDateTime.class, Short.class, Integer.class, Long.class, BigInteger.class, Float.class, Double.class, BigDecimal.class);
```

Comment:
These two lines are too long (a lot longer than the limit introduced in editorconfig).